README.md

```diff
@@ -168,6 +168,12 @@ Output: 8=FIXT.1.1\u00019=66\u000135=A\u000134=1\u000149=NFT2_FIX1\u000156=FGW\u
 
 ## Changes
 
+### 3.3.1
+
+#### Fixed
+
++ Bug when we tried to substitute the values from string extracted from the log if it had content like this `${something}`
```

Comment:
_Bug realated to values like `${something}` in substitute process_
Maybe something like this will be more succinct explanation